app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -432,6 +467,25 @@ const MeetingEditor = () => {
                     )}
                   </ConfirmModal>
                 )}
+                {isEditPage && canDelete && (
+                  <ConfirmModal
+                    title="Hente inn referat fra localstorage?"
+                    message={`Er du sikker på at du vil hente dette møtet? ${localStorage.getItem(
+                      `meeting-${meeting.id}-temp`
```

Comment:
Printing out the report from localstorage makes little sense to the user and gives a bad UI. It's barely readable. Instead, it should say that it has "found" something or that it was last updated at some time x